installing/installing_azure_stack_hub/preparing-to-install-on-azure-stack-hub.adoc

```diff
@@ -0,0 +1,38 @@
+[id="preparing-to-install-on-azure-stack-hub"]
+= Preparing to install on Azure Stack Hub
+include::modules/common-attributes.adoc[]
+:context: preparing-to-install-on-azure-stack-hub
+
+toc::[]
+
+[id="preparing-to-install-on-ash-prerequisites"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+
+[id="requirements-for-installing-ocp-on-ash"]
+== Requirements for installing {product-title} on Azure Stack Hub
+
+Before installing {product-title} on Microsoft Azure Stack Hub, you must configure an Azure account. See xref:../../installing/installing_azure/installing-azure-account.adoc#installing-azure-account[Configuring an Azure account] for details about account configuration, account limits, public DNS zone configuration, required roles, creating service principals, and supported Azure regions.
```

Comment:
This looks like it is pointing to the wrong link, Configuring an Azure Account rather than configuring an Azure Stack Hub account